tools/clang/lib/SPIRV/GlPerVertex.cpp

```diff
@@ -424,15 +424,36 @@ bool GlPerVertex::tryToAccess(hlsl::SigPoint::Kind sigPointKind,
   return false;
 }
 
-uint32_t GlPerVertex::readPosition() const {
+bool GlPerVertex::tryToAccessPointSize(hlsl::SigPoint::Kind sigPointKind,
+                                       llvm::Optional<uint32_t> invocation,
+                                       uint32_t *value, bool noWriteBack) {
+  switch (sigPointKind) {
+  case hlsl::SigPoint::Kind::HSCPIn:
+  case hlsl::SigPoint::Kind::DSCPIn:
+  case hlsl::SigPoint::Kind::GSVIn:
+    *value = readPositionOrPointSize(/*isPosition=*/false);
+    return true;
+  case hlsl::SigPoint::Kind::VSOut:
+  case hlsl::SigPoint::Kind::HSCPOut:
+  case hlsl::SigPoint::Kind::DSOut:
+    writePositionOrPointSize(/*isPosition=*/false, invocation, *value);
+    return true;
+  }
+
+  return false; // Fall back to normal path: GSOut
+}
+
+uint32_t GlPerVertex::readPositionOrPointSize(bool isPosition) const {
   assert(inIsGrouped); // We do not handle stand-alone Position builtin here.
```

Comment:
update comment to
`We do not handle stand-alone Position/PointSize builtin here`

Reply:
Done